integration-tests/clipper_admin_tests.py

```diff
@@ -80,6 +80,18 @@ def test_link_not_registered_model_to_app_fails(self):
             self.clipper_conn.link_model_to_app(app_name, not_deployed_model)
         self.assertTrue("No model with name" in str(context.exception))
 
+    def test_delete_application_correct(self):
```

Comment:
You also need to add this test function to SHORT_TEST_ORDERING list below:
https://github.com/ucbrise/clipper/blob/ff849d4ade612732a72427b156206ec9333073fe/integration-tests/clipper_admin_tests.py#L567

Reply:
Oh good catch!